android/core/src/main/java/com/stadiamaps/ferrostar/core/service/ForegroundNotificationBuilder.kt

```diff
@@ -0,0 +1,66 @@
+package com.stadiamaps.ferrostar.core.service
+
+import android.app.Notification
+import android.app.PendingIntent
+import android.content.Context
+import android.content.Intent
+import java.lang.ref.WeakReference
+import uniffi.ferrostar.TripState
+
+/**
+ * A builder for creating a foreground notification for the Ferrostar navigation service.
+ *
+ * See DefaultForegroundNotificationBuilder in the composeui module for an example/default
```

Comment:
I _think_ this can be linked?

Reply:
Not sure it'll actually work, but worth a shot 👍